doc/releases/changelog-dev.md

```diff
@@ -29,6 +29,27 @@
 
 <h3>Improvements</h3>
 
+* The decorator `self_inverses` now supports all Hermitian Gates.
+  [(#1136)](https://github.com/PennyLaneAI/catalyst/pull/1136)
+
+  The full list of supported gates are as follows:
+
+  Unary Gates:
+  - [`qml.Hadamard`](https://docs.pennylane.ai/en/stable/code/api/pennylane.Hadamard.html)
+  - [`qml.PauliX`](https://docs.pennylane.ai/en/stable/code/api/pennylane.PauliX.html)
+  - [`qml.PauliY`](https://docs.pennylane.ai/en/stable/code/api/pennylane.PauliY.html)
+  - [`qml.PauliZ`](https://docs.pennylane.ai/en/stable/code/api/pennylane.PauliZ.html)
+
+  Binary Gates:
+  - [`qml.CNOT`](https://docs.pennylane.ai/en/stable/code/api/pennylane.CNOT.html)
+  - [`qml.CY`](https://docs.pennylane.ai/en/stable/code/api/pennylane.CY.html)
+  - [`qml.CZ`](https://docs.pennylane.ai/en/stable/code/api/pennylane.CZ.html)
+  - [`qml.SWAP`](https://docs.pennylane.ai/en/stable/code/api/pennylane.SWAP.html)
+
+  Trinary Gates: Toffoli
+  - [`qml.Toffoli`](https://docs.pennylane.ai/en/stable/code/api/pennylane.Toffoli.html)
```

Comment:
Should these be listed as "single-/two-/three-qubit gates" instead? I think that would follow more standard nomenclature. When I read "unary/binary/ternary" I think of the number of states a bit can be in, not the number of bits that are inputs to the gate.

Reply:
(Also I just checked and the original story used the "one/two/three-bit" wording)

Reply:
Thanks for the comment. I don't have a strong preference on the naming. changed it to one-bit, two-bit,...